Scripts/script-CommonServer.yml

```diff
@@ -203,6 +203,18 @@ script: |-
       return mdResults;
   }
 
+  var tblToMd = tableToMarkdown;
+
+  /**
+   * Converts underscore case strings to camel case
+   * @param {String} string - string to be converted - i.e. hello_world
+   * @returns {String} - converted string - i.e. HelloWorld
+  */
+  var undrscrToCamelCase = function(string){
+      string = '_'+string;
```

Comment:
In JavaScript (and other languages), it is not a good practice to mutate the function input, so please change the assignment to another variable.